packages/server/src/koaJsonRpc/index.ts

```diff
@@ -119,6 +119,8 @@ export default class KoaJsonRpc {
         return;
       }
 
+      this.logger.trace(`[Request ID: ${this.getRequestId()}] Request body ${JSON.stringify(body)}`);
```

Comment:
This may result in repeated info in the logs as some methods log the contents of the body which are params.
In a few cases that's transaction bytes or a whole contract.
So we might need some coordination on that

Reply:
@Neurone Is this still an issue?

Reply:
The code of the PR must be probably updated at this point, but yes, imho the issue this PR tries to solve is still there.

I think when as a developer I set TRACE as a log level, I want to see anything because I'm doing this for heavy debugging activities. The PR includes the answers and the responses bodies to the logs, and this info is essential for debugging purposes.

At the same time, I think the default log level should be INFO, not TRACE as we use now. In general I think using INFO as default is a good practice, and in this case it shows another good reason why. If we use TRACE as our default, when adding more data to the log it **seems** too much (as reported by Nana) but the problem for me it's not there, because when I enable trace I want to see everything, and duplicated data are not an issue at all and in some cases are very useful (because I can see nothing happened to those data during the business flow).

Reply:
Thank you for your efforts on this PR and for sharing your insights on the logging levels. We appreciate your initiative to enhance our debugging process.

After discussing with the team, we've decided that it's best to keep the default log level at TRACE for now. Given the current stage of the project, having detailed logs by default is helpful for our ongoing development and debugging activities.

We agree that setting the default log level to INFO is a good practice for more mature projects. We'll certainly revisit this topic as the project evolves.

For the time being, I am planning to close this PR.  Let me know if you have any questions or concerns.